doc/mgmt/SONiC_Management_Framework_sFlow.md

```diff
@@ -161,26 +170,26 @@ Syntax:
 
 port# : [0 - 65535] (default is: 6343)
 
-`sflow collector <collector name> <IPv4/IPv6 address> [port #]`
+`sflow collector <IPv4/IPv6 address> [port #]`
 
 ```
-sonic(config)# sflow collector col1 1.1.1.1
+sonic(config)# sflow 1.1.1.1
```

Comment:
Still need the "collector" keyword here.

Reply:
Will fix. Thank you.